services/storage/tests/unit/test_handlers_files.py

```diff
@@ -873,6 +873,128 @@ async def test_download_file(
     assert filecmp.cmp(uploaded_file, dest_file)
 
 
+@pytest.mark.parametrize(
+    "file_size",
+    [
+        pytest.param(parse_obj_as(ByteSize, "1")),
+    ],
+    ids=byte_size_ids,
+)
+async def test_download_file_cases(
```

Comment:
why do you repeat the test from above? case 2 is exactly what is done above
why not have:
- test_download_file_in_directory
- test_download_missing_file_raises
- test_download_file_with_no_access_raises

This would be easier to maintain and understand I think. this test now kinds of is a huge one that covers everything